packages/conventional-changelog-writer/test/index.spec.js

```diff
@@ -4,10 +4,12 @@ import { delay, throughObj } from '../../../tools/test-tools.js'
 import conventionalChangelogWriter, { parseArray } from '../index.js'
 
 // sv-SEis used for yyyy-mm-dd format
-const dateFormatter = Intl.DateTimeFormat('sv-SE', {
-  timeZone: 'UTC'
-})
-const today = dateFormatter.format(new Date())
+function dateFormatter(timeZone) {
+  return Intl.DateTimeFormat('sv-SE', {
+    timeZone
+  })
+}
+const today = (timeZone = 'UTC') => dateFormatter(timeZone).format(new Date())
```

Comment:
```suggestion
function formatDate(date, timeZone = 'UTC') {
  // sv-SE is used for yyyy-mm-dd format
  return Intl.DateTimeFormat('sv-SE', {
    timeZone
  }).format(date)
}

function getTodayDate(timeZone) {
  return formatDate(new Date(), timeZone)
}

const todayUtc = getTodayDate()
```

lets reduce functions calls, please use `todayUtc` value instead of `today()` calls

Reply:
Fixed based on your advice!